examples/helloElektra.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include <kdb.h>
+
+#include <stdio.h>
+
+int main (void)
+{
+	KeySet * config = ksNew (0, KS_END);
+	Key * root = keyNew ("system/test", KEY_END);
+
+	printf ("Open key database\n");
+	KDB * handle = kdbOpen (root);
+
+	printf ("Retrieve key set\n");
+	kdbGet (handle, config, root);
+
+	printf ("Number of mappings: %zu\n", ksGetSize (config));
```

Comment:
Is it clear what "mappings" are?

Reply:
I replaced the term “mapping“ with “key-value“ pairs in the latest update.